src/main/java/org/sagebionetworks/web/client/widget/table/explore/TableEntityPlotsWidget.java

```diff
@@ -0,0 +1,671 @@
+package org.sagebionetworks.web.client.widget.table.explore;

```

Comment:
* Initially copied from TableEntityWidget with the following modifications.
 * Uses SRC QueryWrapperPlotNav
 * Removed Query Input widget.  Removed concept of simple and advanced search.
 * This is responsible for editing table rows.
 * Copied isQueryResultEditable() from TableQueryResultWidget.